.coveragerc

```diff
@@ -15,8 +15,8 @@ omit =
         hummingbot/client/ui/parser.py
         hummingbot/connector/connector/balancer*
         hummingbot/connector/connector/terra*
-        hummingbot/connector/connector/uniswap*
```

Comment:
Is it needed here? I think that's connector not strategy

Reply:
Ok